app/models/manageiq/providers/openstack/cloud_manager/provision_workflow.rb

```diff
@@ -91,4 +91,5 @@ def dialog_name_from_automate(message = 'get_dialog_name')
   def self.provider_model
     ManageIQ::Providers::Openstack::CloudManager
   end
+  private_class_method :provider_model
```

Comment:
yes, needs to be public, it's called from app/models/miq_provision_virt_workflow.rb